src/app/Ranking.js

```diff
@@ -45,6 +45,9 @@ export class Ranking {
   }
 
   getScores() {
-    return JSON.parse(localStorage.getItem(this.mode)) ?? [];
+    const results = JSON.parse(localStorage.getItem(this.mode));
+    return results
```

Comment:
Możesz to też zapisać tak:

```
return results?.sort((a, b) => a.score / a.maxScore - b.score / b.maxScore) ?? [];
```

Albo tak:
```
```
return (results ?? []).sort((a, b) => a.score / a.maxScore - b.score / b.maxScore) ?? [];
```
```

Reply:
niezłe!